.gitignore

```diff
@@ -13,6 +13,8 @@ local.properties
 .settings/
 .loadpath
 .recommenders
+kami_Client.launch
+kami_Server.launch
```

Comment:
These still need to go, as the `*.launch` exemption covers these.

Reply:
Thought I deleted them, let me look into it.